modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteClusterSnapshotDeltaTest.java

```diff
@@ -77,7 +80,8 @@ public static Collection<Object[]> parameters() {
     @Test
     public void testSendDelta() throws Exception {
         int keys = 10_000;
-        byte[] payload = new byte[DFLT_PAGE_SIZE / 2];
+        int pageSz = IgniteSystemProperties.getInteger(IGNITE_DEFAULT_DATA_STORAGE_PAGE_SIZE, DFLT_PAGE_SIZE);
```

Comment:
AbstractSnapshotSelfTest#PAGE_SIZE ?

Reply:
No, it cannot be used here, because PAGE_SIZE doesn't contain valid value

Reply:
Oh, I forgot  that i had rewritten it already :)